### PR TITLE
added version flag to consul-k8s cli docs

### PR DESCRIPTION
**Pull request by trujillo-adam**
In this PR, I added the `version` subcommand. I also added `--help` flags to the `install`/`uninstall` commands

### CODE REVIEW

**Review by david-yu — APPROVED** [review]
This looks good! We probably will need to change the example output of the version when this goes GA. 

 bin/consul-k8s version
    consul-k8s 0.35.0 (or something like that) without the dev

File: website/content/docs/k8s/k8s-cli.mdx [hunk 1]
```diff
@@ -27,6 +27,13 @@ This topic describes the commands, subcommands, and available options for using
    mv bin/consul-k8s /usr/local/bin/consul-k8s
    ```
 
+1. (Optional) Issue the `consul-k8s version` command to verify the installation:
+
+   ```shell-session
+   bin/consul-k8s version
+   consul-k8s 0.34.1-dev
+   ```
```
Comment (lkysow) [inline website/content/docs/k8s/k8s-cli.mdx:35]: If you've already `mv`'d it then `bin/consul-k8s` will fail so I think we just need the second line?

**Review by lkysow — APPROVED** [review]

File: website/content/docs/k8s/k8s-cli.mdx (hunk 1 above)
Comment (trujillo-adam) [reply website/content/docs/k8s/k8s-cli.mdx:35]: ah, thanks, good catch

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/463826.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 937fd1004171585a07484e77e5fb3e416d01691c onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 937fd1004171585a07484e77e5fb3e416d01691c onto `release/1.10.x` succeeded!

**Pull request merged**